### ISSUE TITLE
Changes of README.md without concent

### ISSUE THREAD

**Issue by alexandre1985** [slapperwan/gh4a#991]
I just noticed that OctoDroid changes this README.md text, from:
```
<a href="SomeRepositoryFilename.js" ...></a>
```
to
```
<a href="https://raw.github.com/myUserName/repoName/master/SomeRepositoryFilename.js" ...></a>
```

The motives for this reformatation are not clear to me.

I don't know about others, but I am not a newbie programmer that may not know how to write READMEs, so I want this re-formatting feature to be turned off, by default, on all people that are reading my README's on Octodroid.  

Disabling this OctoDroid feature is a suitable option too.

**Comment by maniac103** [comment]
Just to be clear, we're talking about the readme shown on the repository overview, right? If so, the rationale for this rewriting is that the content is not displayed in a WebView / browser, so there's no 'page context' around that knows about the page base URL. Thus any images or hyperlinks in the document need to be absolute, not relative.

If this feature were turned off, images wouldn't load and URLs clicked would point to nowhere. I doubt that's what you (or any other user) wants.

What's the actual problem with the URL rewrite?

**Comment by maniac103** [comment]
OK, found the problem from your recent commits. I'd summarize it as 'only actual href tags should be rewritten, hrefs in code or quote blocks shouldn't'...would you agree to that?

**Comment by alexandre1985** [comment]
I agree that displayed *text* uri (versus image/video display) should, in my opinion, stay as is written (in README.md file)

**Comment by maniac103** [comment]
Can't do it for image URLs only ... when clicking a link, the link is opened in the browser, and the browser doesn't support opening the page `somedirectory/somepage.php` for obvious reasons.

**Comment by alexandre1985** [comment]
@maniac103 I said, every URI except image URI and video URI

**Comment by maniac103** [comment]
> In my opinion, no re-formatting must be done

¯\_(ツ)_/¯ Sadly, it doesn't work that way ... see [the comments](https://github.com/slapperwan/gh4a/issues/991#issuecomment-647147462) [above](https://github.com/slapperwan/gh4a/issues/991#issuecomment-647164265) for the explanation.

Keeping this open for the 'shouldn't rewrite URLs in quote or code blocks' issue though.

**Comment by alexandre1985** [comment]
I saw the comments, and I understand.
Also, there is **no** relation to displayed **text** uris, such as `<a href="myjs.js" ... ></a>` code text.
This is what I meant by **text** uri. In the text uris there is not justification for re-formatting `..href="myjs.js" ...` into `...href="https://raw.github.com/...myjs.js"`

**Comment by alexandre1985** [comment]
Is this on consideration?

**Comment by maniac103** [comment]
Yes ... That's why it's open. It's a big effort though, so don't hold your breath.

**Comment by alexandre1985** [comment]
Thank you man. I'll keep this app. :guardsman:
